Include/httpClient/mock.h

```diff
@@ -118,11 +120,20 @@ STDAPI HCMockRemoveMock(
     );
 
 /// <summary>
-/// Removes and cleans up all mock calls added by HCMockAddMock.
+/// Removes all mock calls added by HCMockAddMock.
 /// </summary>
 /// <returns>Result code for this API operation.  Possible values are S_OK, or E_FAIL.</returns>
 STDAPI HCMockClearMocks() noexcept;
 
+/// <summary>
+/// Closes a mock HTTP call handle.
+/// </summary>
+/// <param name="call">The handle of the mock HTTP call.</param>
+/// <returns>Result code for this API operation.  Possible values are S_OK, E_INVALIDARG, or E_FAIL.</returns>
+STDAPI HCMockCallCloseHandle(
+    _In_ HCMockCallHandle call
+    ) noexcept;
```

Comment:
Maybe worth adding HCMockCallDuplicateHandle for completeness?

Reply:
added